frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.styled.tsx

```diff
@@ -25,4 +25,7 @@ export const Label = styled.span`
 export const BoldExternalLink = styled(ExternalLink)`
   color: var(--mb-color-brand);
   font-weight: bold;
+  display: flex;
```

Comment:
This is needed to space the new icon nicely. This icon wasn't here previously.
![image](https://github.com/user-attachments/assets/322347e5-4153-4946-969a-b3db47ec85a8)